lua/telescope/builtin/init.lua

```diff
@@ -859,6 +859,60 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.registers = function(opts)
+  opts = opts or {}
+
+  local registers_table = {}
+  table.insert(registers_table, {kind = "Unnamed",      key = "\""})
+  table.insert(registers_table, {kind = "Small Delete", key = "_"})
+  table.insert(registers_table, {kind = "Alternate",    key = "#"})
+  table.insert(registers_table, {kind = "Expression",   key = "="})
+  table.insert(registers_table, {kind = "Black Hole",   key = "_"})
+  table.insert(registers_table, {kind = "Last Search",  key = "/"})
+
+  -- clipboaord
+  local selection = {"*", "+"}
+  for i in pairs(selection) do
+    table.insert(registers_table, {kind = "Selection",    key = selection[i]})
+  end
+
+  -- named
+  for i = 0, 9 do
+    table.insert(registers_table, {kind = "Numbered", key = tostring(i)})
+  end
+
+  -- alpha
+  for i = 65, 90 do
+    table.insert(registers_table, {kind = "Named", key = string.char(i)})
+  end
+
+  -- readonly
+  local readonly = {":", ".", "%"}
+  for i in pairs(readonly) do
+    table.insert(registers_table, {kind = "Read-Only",    key = readonly[i]})
+  end
+
+  pickers.new(opts,{
+    prompt_title = 'Registers',
+    finder = finders.new_table {
+      results = registers_table,
+      entry_maker = make_entry.gen_from_registers(opts),
+    },
+    previewer = previewers.vimgrep.new(opts),
```

Comment:
Does this work for you? I think you might have to remove that line and not enable a previewer at all

Reply:
There's currently no previewer at all. I'd disabled it in my `require` call.

For most 90% of entries, the preview is going to be the same one-line as the register content (In my workflow anyway), the latter 10% you may well want to see the entire content.

The only other (silly) idea  I had, was to copy the original buffer, and preview what the buffer would look like _after_ the paste/action.

I think it will  just be preview-less.

Reply:
Yeah i agree that we don't need a previewer. I just thought this previewer might be broken because it expects a `filename` entry.
I just tested it and i got just a bat error. So i suggest removing that line, so people with a previewer configured will not be confused about that error.